frontend/src/fixtures/ucsbSubjectsFixtures.js

```diff
@@ -0,0 +1,43 @@
+const ucsbSubjectsFixtures = {
+    oneSubject: {
+        "id": 1,
+        "subjectCode": "test1",
+        "subjectTranslation": "test1",
+        "deptCode": "test1",
+        "collegeCode": "test1",
+        "relatedDeptCode": "test1",
+        "inactive": false
+    },
+    threeSubjects: [
+      {
+        "id": 1,
+        "subjectCode": "test1",
+        "subjectTranslation": "test1",
+        "deptCode": "test1",
+        "collegeCode": "test1",
+        "relatedDeptCode": "test1",
+        "inactive": false
+      },
+      {
+        "id": 2,
+        "subjectCode": "test2",
+        "subjectTranslation": "test2",
+        "deptCode": "test2",
+        "collegeCode": "test2",
+        "relatedDeptCode": "test2",
+        "inactive": false
+      },
+      {
+        "id": 3,
+        "subjectCode": "test3",
+        "subjectTranslation": "test3",
+        "deptCode": "test3",
+        "collegeCode": "test3",
+        "relatedDeptCode": "test3",
+        "inactive": false
+      }
+    ]
+};
```

Comment:
Please considering use real data from the UCSB Subjects API in your fixtures instead of fake data like this.